src/NativeBase/Containers/ProjectSearchContainer.tsx

```diff
@@ -1,66 +1,43 @@
-/**
- * @file Projects search screen container.
- */
-
 import React, { useState } from 'react'
-import Fuse from 'fuse.js' // fuzzy text search - see docs at https://fusejs.io
 import { ScrollView, VStack } from 'native-base'
 import { useSelector } from 'react-redux'
-import { ListRouteParams, ListSearch, ListType } from './ListContainer'
 import ChoicesList, {
   ChoicesListChoice,
   ChoicesListColour,
   ChoicesListFontStyle,
 } from '../Components/ChoicesList'
 import FreeSearchBar from '../Components/FreeSearchBar'
-import SegmentedPicker, {
-  SegmentedPickerOption,
-} from '../Components/SegmentedPicker'
+import TagButtons from '../Components/TagButtons'
 import { navigate, RootStackParamList } from '@/Navigators/utils'
 import {
-  Projects,
   ProjectsSearchField,
   ProjectSector,
   ProjectTechnology,
+  Projects,
 } from '@/Services/modules/projects'
+import { ProjectsState } from '@/Store/Projects'
+import { searchByArray, fuzzySearchByArray } from '@/Utils/Search'
 import {
-  roleGroups,
   RoleGroupName,
+  roleGroups,
 } from '@/Services/modules/projects/roleGroups'
-import { ProjectsState } from '@/Store/Projects'
-import { searchByArray, fuzzySearchByArray } from '@/Utils/Search'
-
-enum Tab {
```

Comment:
i would have kept the enum, as we would use only the enums throughout this code and if we want to change a name to another one we'd only change it once here.